runner/runner.py

```diff
@@ -64,21 +61,18 @@ def append_id(block, pipeline_id: str):
 class Runner:
     def __init__(
         self,
-        run_config: RunConfig,
-        pipeline: Pipeline,
+        experiment: Experiment,
         data: Dict[str, Union[pd.Series, List]],
         labels: pd.Series,
-        evaluators: Evaluators,
         plugins: List[Optional[Plugin]],
     ) -> None:
-        self.config = run_config
+        self.experiment = experiment
         self.run_path = f"{Const.output_runs_path}/{datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}/"
-        self.pipeline = pipeline
         self.store = Store(data, labels, self.run_path)
-        self.evaluators = evaluators
         self.plugins = obligatory_plugins + plugins
 
-        self.pipeline = overwrite_model_configs(self.config, self.pipeline)
+        self.pipeline = experiment.pipeline  # maybe we want to deepcopy it first?
```

Comment:
question: should we deepcopy the pipeline first before we modify it?

Reply:
I think you already may have asked this question

Reply:
It can be deepcopied, although it won't make a difference inpractice because with each run we run the function aswell.